packages/steveo/src/index.ts

```diff
@@ -70,8 +80,10 @@ export class Steveo implements ISteveo {
       | RedisConfiguration
       | SQSConfiguration
       | DummyConfiguration,
+    storage: Storage,
```

Comment:
I would put this at the end with a default Storage engine that throws. Means we can upgrade to whatever version this is without having to adopt workflows, and then when we want to use workflows we actually hook up storage

Reply:
Can do

Reply:
Done https://github.com/ordermentum/steveo/pull/800/commits/f2a9489ac71131df8663de678ef7aef1b3492461